mkdocs.yml

```
@@ -54,12 +54,11 @@ pages:
   - APIv4:
     - APIv4 Usage: api/v4/usage.md
     - APIv4 Actions: api/v4/actions.md
-  #  - APIv4 Options: api/v4/options.md
     - APIv4 Joins: api/v4/joins.md
     - APIv4 Chaining: api/v4/chaining.md
     - APIv4 Custom Data: api/v4/custom-data.md
-  #  - APIv4 Examples: api/v4/examples.md
-  #  - APIv4 Changes: api/v4/changes.md
+    - Differences between Api v3 and v4: api/v4/differences-with-v3.md
```

Can we move this to the "root" level of the API folder and drop the v4 part of the name. e.g:

```suggestion
    - Differences Between API v3 and v4: api/differences-between-API-v3-v4.md
```

---

IMO this section is fine where it is because it's really about v4 and how to upgrade from previous versions; it's not about the api in general.

---

That's fair - still need to correct for "Title" case in the page title maybe? Otherwise I'm good with this!